tests/python/pants_test/backend/jvm/tasks/test_ivy_resolve_integration.py

```diff
@@ -19,6 +19,7 @@ def test_java_compile_with_ivy_report(self):
     with self.temporary_workdir() as workdir:
       ivy_report_dir = '{workdir}/ivy-report'.format(workdir=workdir)
       pants_run = self.run_pants_with_workdir([
+          '--resolver-resolver=ivy',
```

Comment:
I think adding this flag to all of the tests in this suite makes them a little harder to scan.

How would you feel about moving that flag into an override of `run_pants_with_workdir`?

Reply:
good idea. extracted.